README.md

```diff
@@ -1,42 +1,29 @@
-# Syne Tune
+# Syne Tune: Large-Scale and Reproducible Hyperparameter Optimization
 
 [![Release](https://img.shields.io/badge/release-0.2-brightgreen.svg)](https://pypi.org/project/syne-tune/)
 [![Python Version](https://img.shields.io/badge/3.7%20%7C%203.8%20%7C%203.9-brightgreen.svg)](https://pypi.org/project/syne-tune/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Downloads](https://pepy.tech/badge/syne-tune/month)](https://pepy.tech/project/syne-tune)
 
-This package provides state-of-the-art distributed hyperparameter optimizers (HPO) where trials 
- can be evaluated with several trial backend options (local backend to evaluate trials locally;
-SageMaker to evaluate trials as separate SageMaker training jobs; a simulation backend to quickly benchmark parallel 
-asynchronous schedulers).
+![Alt Text](docs/synetune.gif)
+
+This package provides state-of-the-art distributed hyperparameter optimizers (HPO) with the following key features:
+* wide coverage of HPO optimizers including more than 20 optimizer methods supporting asynchronous optimization with multiple workers including:
```

Comment:
Maybe we could improve the wording here. For example: 'wide coverage (>20) of different HPO methods for asynchronous optimization with multiple workers, including:'